### ISSUE TITLE
[SF][A11y][FlexibleColumnLayout] accessibilityTexts does not work

### ISSUE THREAD

**Issue by 2bno1** [SAP/ui5-webcomponents-react#2544]
**Describe the bug**
I'm not able to customize the accessbile name of regions by setting `accessibilityTexts` prop of `FlexibleColumnLayout`

**Isolated Example**
https://codesandbox.io/s/flex-column-layout-t9z68?file=/src/App.js:392-410

**Expected behavior**
I can customize the accessible name of regions by `accessibilityTexts`

**Screenshots**
![WeChat Image_20220201100633](https://user-images.githubusercontent.com/1751846/151903215-a47e95ca-d799-4f56-90d6-46d285b68f28.png)
As seen from the screenshot, I expect to change the accessible name of first region, but it's still the default text: 'First Column'.

**UI5 Web Components for React Information**
`@ui5/webcomponents` version: latest
`@ui5/webcomponents-react` version: latest
Operating System: Windows 10 
Browser: Chrome

**Comment by MarcusNotheis** [comment]
Hey @2bno1,

thanks for reporting.
As far as I know, the UI5 Web Components don't support JSON objects as attributes, so using `accessibilityTexts` as prop will never reach the DOM. You have to add this object via JavaScript, I would recommend using `useEffect` for that:
https://codesandbox.io/s/flex-column-layout-forked-hlo9e?file=/src/App.js

I'll also create a follow up issue on the UI5 Web Components side to get the docs (and as a consequence our prop types) fixed.
